dspace/config/ldn/announce-relationship

```diff
@@ -46,7 +47,7 @@
     "type": "Relationship"
   },
   "origin": {
-    "id": "${params[0]}",
+    "id": "${params[14]}",
```

Comment:
Is there a reason why a new parameter was added even though from the description above I see that the parameter is equal to parameter 0? I think it's probably a typo in the description of this file and the parameter is not always intended to be equal to parameter 0. In this case a better description of the parameter might be needed

Reply:
The description needs amending as in this instance this parameter (14) will always use the DSpace UI URL; whereas parameter 0 can be either a person actor (submitter email) OR the DSpace UI URL (service actor) depending on the service requirements.

Reply:
Amended now - the announce-relationship and request-ingest templates do not need this parameter as the config will always be the service actor (dspace.ui.url).